packages/static-hosting/lib/static-hosting.ts

```diff
@@ -610,12 +612,12 @@ export class StaticHosting extends Construct {
     }
   }
 
-  private createRemapBehavior(from: string, to: string): EdgeLambda[] {
+  private createRemapBehavior(from: string, to?: string): EdgeLambda[] {
     const lambdas: EdgeLambda[] = [];
 
     // If the remap is to a different path, create a Lambda@Edge function to handle this
     // Remove special characters from path
-    if (from.replace(/\*$/, "") !== to) {
+    if (from.replace(/\*$/, "") !== to && to) {
```

Comment:
Can you flip these conditions around so it checks if `to` is falsey first?

Reply:
Yep, done.